### PR TITLE
Add graphQL related docs and fixed some issues 

### PR DESCRIPTION
**Pull request by HiranyaKavishani**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx [hunk 1]
```diff
@@ -309,80 +304,90 @@ class APICreateGraphQL extends React.Component {
...
+                                )}
+                            </div>
+                            <div>
+                                <div>
+                                    <div>
+                                        <Button
+                                            disabled={activeStep === 0}
+                                            onClick={this.handleBack}
+                                            className={classes.button}
+                                        >Back
```
Comment (TanyaM) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx]: Need to localize the string

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx [hunk 2]
```diff
@@ -309,80 +304,90 @@ class APICreateGraphQL extends React.Component {
...
+                                            variant='contained'
+                                            color='primary'
+                                            onClick={this.handleNext}
+                                            className={classes.button}
+                                            disabled={
+                                                (valid.graphQLFile.invalidFile && uploadMethod === 'file')
+                                            }
+                                        >
+                                            {activeStep === steps.length - 1 ? (
+                                                'Finish'
```
Comment (TanyaM) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx]: Need to lcoalize

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/data/api.js [hunk 3]
```diff
@@ -238,6 +238,9 @@ class API extends Resource {
     }
 
     getProductionEndpoint() {
+        if(!this.endpointConfig){
```
Comment (TanyaM) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/data/api.js]: formatting issue

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx (hunk 1 above)
Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx]: This file is no longer used as the code had been refactored with react hooks 
https://github.com/wso2/carbon-apimgt/pull/6944/files
Therefore, we don't want to add localization at here

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx (hunk 2 above)
Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx]: This file is no longer used as the code had been refactored with react hooks 
https://github.com/wso2/carbon-apimgt/pull/6944/files
Therefore, we don't want to add localization at here

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/data/api.js (hunk 3 above)
Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/data/api.js]: Fixed with https://github.com/wso2/carbon-apimgt/commit/0bd3176e5443682dc5b83817d4446dcdb9797b54#diff-e2b0ed5829539cacc4b0e53b3e140cb9R255

**Pull request merged**